lib/manageiq/appliance_console/cli.rb

```diff
@@ -20,7 +20,7 @@ def host
 
     # database hostname
     def hostname
-      options[:internal] ? "localhost" : options[:hostname]
+      options[:standalone] || options[:internal] ? "localhost" : options[:hostname]
```

Comment:
I would rather see this change made in the definition of `#database?`

Instead of making `#hostname` be truthy when `--standalone` is specified, I think it makes more sense to make `#database` truthy when we are given a hostname or when `--standalone` is specified.

Reply:
Sounds good!